internal/builder/packages.go

```diff
@@ -150,6 +173,18 @@ func BuildPackage(options BuildOptions) (string, error) {
 		return "", errors.Wrap(err, "clearing package contents failed")
 	}
 
+	m, err := packages.ReadPackageManifestFromPackageRoot(options.PackageRoot)
+	if err != nil {
+		return "", errors.Wrapf(err, "reading package manifest failed (path: %s)", options.PackageRoot)
+	}
+	if license := m.Source.License; license != "" {
+		logger.Debugf("Write license text for %q", license)
+		err := licenses.WriteTextToFile(license, filepath.Join(destinationDir, "LICENSE.txt"))
```

Comment:
Will LICENSE.txt be defined in the package-spec?

Reply:
I hadn't considered it, but it could very well be there, I will open a PR.

Reply:
https://github.com/elastic/package-spec/pull/367